Add mounts to be able to correctly deploy a Next.js application

We should make the following work without errors:

```console
npx create-next-app@latest
platformify
git commit -am 'Platformify'
platform push
```

Seems like Node.js needs the following mounts to operate correctly (they need to be verified):

```yaml
mounts:
    '/.npm/_logs':
        source: local
        source_path: npm-logs
    '/.npm/_cache':
        source: local
        source_path: npm-cache
    '/.next':
        source: local
        source_path: 'next'
```

We might need to move building to the `deploy` step, to ensure the files needed by Next.js exist.

---

## Original description

I tested it this with a `next.js` app this morning by running `npx create-next-app@latest` and the `project:init` command. Deploying this to a PSH project without any change throws a `502` with:

```
npm ERR! Log files were not written due to an error writing to the directory: /app/.npm/_logs
npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
npm WARN logfile could not create logs-dir: Error: ENOENT: no such file or directory, mkdir '/app/.npm/_logs'
npm WARN logfile could not be created: Error: ENOENT: no such file or directory, open '/app/.npm/_logs/2023-04-26T06_22_26_196Z-debug-0.log'
npm ERR! code EROFS
npm ERR! syscall mkdir
npm ERR! path /app/.npm/_cacache
npm ERR! errno -30
npm ERR! rofs EROFS: read-only file system, mkdir '/app/.npm/_cacache'
npm ERR! rofs Often virtualized file systems, or other file systems
npm ERR! rofs that don't support symlinks, give this error.
```

so I had to add at least `.npm/_cacache` as a mount for it to work (probably `.npm/_logs` also makes sense)

--- a/platformifiers/templates/nextjs/.platform.app.yaml
+++ b/platformifiers/templates/nextjs/.platform.app.yaml
@@ -100,6 +100,17 @@ web:
   {{ end }}
 
 ##########################
+# The size of the persistent disk of the application (in MB).
+disk: 512
+
+# The mounts that will be available to your application when deployed.
+mounts:
+    # NPM caching directory, so it must be writeable.
+    '/.npm':
+        source: local
+        source_path: 'npm'
+
+##########################
 # Services
 
 # The relationships of the application with services or other applications.

